doc/source/user_guide/fields/field_data.rst

```diff
@@ -147,98 +161,37 @@ First, create a transaction object for field data.
   >>> transaction = solver.fields.field_data.new_transaction()
   # This creates a new transaction object for batching multiple requests.
 
-Then combine requests for multiple fields using ``add_<items>_request`` methods in a single transaction:
-
-- ``add_surfaces_request`` adds a surfaces request.
-- ``add_scalar_fields_request`` adds a scalar fields request.
-- ``add_vector_fields_request`` adds a vector fields request.
-- ``add_pathlines_fields_request`` adds a pathlines fields request.
+Then combine requests for multiple fields using ``add_requests`` methods in a single transaction
+and call the ``get_response`` method to execute the transaction and retrieve the data.
 
 .. code-block:: python
 
-  >>> transaction.add_surfaces_request(
-  >>>     surfaces=[1], data_types=[SurfaceDataType.Vertices, SurfaceDataType.FacesCentroid]
-  >>> )
-  # Adds a request for surface data such as vertices and centroids.
-  >>> transaction.add_scalar_fields_request(
-  >>>     surfaces=[1, 2], field_name="pressure", node_value=True, boundary_value=True
-  >>> )
-  # Adds a request for scalar field data like pressure.
-  >>> transaction.add_vector_fields_request(
-  >>>     surfaces=[1, 2], field_name="velocity"
-  >>> )
-  # Adds a request for vector field data like velocity.
-
-You can call the ``get_fields`` method to execute the transaction and retrieve the data:
+  >>> su1 = SurfaceFieldDataRequest(surfaces=[1], data_types=[SurfaceDataType.Vertices, SurfaceDataType.FacesCentroid])
```

Comment:
@prmukherj  Feel free to us longer, more descriptive names for the request objects.

Reply:
Done, Thank you.